src/pytest_servers/local.py

```diff
@@ -1,6 +1,6 @@
 import pathlib
 
-from fsspec.implementations.local import LocalFileSystem
+from dvc_objects.fs.implementations.local import LocalFileSystem
```

Comment:
`dvc-data` can monkeypatch it if needed. We should not make pytest-servers depend on `dvc_objects.fs`, it should provide pure-fsspec filesystems. 

Note that it may also fail on `S3Path` as it's a `s3fs` filesystem, not 	`dvc_objects.fs.S3FileSystem